esdb/client.go

```diff
@@ -434,6 +434,28 @@ func (client *Client) ConnectToPersistentSubscription(
 	)
 }
 
+func (client *Client) ConnectToPersistentSubscriptionToAll(
```

Comment:
The RFC calls this method `SubscribeToPersistentSubscriptionToAll`

https://github.com/EventStore/architecture-and-planning/blob/main/rfcs/022%20-%20gRPC%20Persistent%20Subscriptions.md#supported-operations

Reply:
Oh, just saw Yves's comment, ignore me.

Reply:
Yeah, also needs to be changed in `ConnectToPersistentSubscription`, so good to do a single alignment PR.